test/e2e/feature/feature.go

```diff
@@ -357,6 +357,10 @@ var (
 	// (used for testing specific log stream <https://kep.k8s.io/3288>)
 	PodLogsQuerySplitStreams = framework.WithFeature(framework.ValidFeatures.Add("PodLogsQuerySplitStreams"))
 
+	// Owner: sig-node
+	// Marks tests that require a cluster with PodObservedGenerationTracking
+	PodObservedGenerationTracking = framework.WithFeature(framework.ValidFeatures.Add("PodObservedGenerationTracking"))
+
```

Comment:
This might not be needed, depending on whether https://github.com/kubernetes/kubernetes/pull/130580 gets merged.

Reply:
ack, I'll keep an eye on it. Thanks!